3p/integration.js

```diff
@@ -361,8 +361,8 @@ function triggerResizeRequest(width, height) {
   nonSensitiveDataPostMessage('embed-size', {width, height});
 }
 
-function triggerRenderStart(width, height) {
-  nonSensitiveDataPostMessage('render-start', {width, height});
+function triggerRenderStart(opt_data) {
+  nonSensitiveDataPostMessage('render-start', {opt_data});
```

Comment:
Hmm just found I can retrieve by `data.width` instead of `data.opt_data.width`. Either way will work then